### PR TITLE
feat: market info - price monitoring bounds, calc 24h volume, oracle price & termination IDs etc.

### LINKED ISSUE #994: Market info covering missing details (price monitoring bounds, 24h volume etc.)
Few things left to be added:

- [x] total fees added in https://github.com/vegaprotocol/frontend-monorepo/pull/889
- [x] 24 hour volume
- [x] insurance pool?
- [x] price monitoring bounds ?
- [x] oracle details to be done in https://github.com/vegaprotocol/frontend-monorepo/issues/626
Technical:  in-memory cache for tradable instrument

_Originally posted by @MadalinaRaicu in https://github.com/vegaprotocol/frontend-monorepo/issues/625#issuecomment-1210509057_

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #994 

# Description ℹ️

Add a few missing details from market information such as 

- [x] price monitoring bounds
- [x] 24h volume
- [x] open interest
- [x] insurance pool balance
- [x] oracle (price & termination IDs + oracleSpecBinding)

# Demo 📺

image
image
image

### CODE REVIEW

File: apps/trading-e2e/src/integration/market-info.cy.ts [hunk 1]
```diff
@@ -25,19 +25,13 @@ describe('market info is displayed', () => {
 
   it('market volume displayed', () => {
     cy.getByTestId(marketTitle).contains('Market volume').click();
-
-    validateMarketDataRow(0, 'Indicative Volume', '0');
```
Comment (MadalinaRaicu) [inline apps/trading-e2e/src/integration/market-info.cy.ts:29]: removing some tests - best bid volumes might be taken out of market info soon

**Review by mattrussell36 — APPROVED** [review]

File: libs/deal-ticket/src/components/info-market.tsx [hunk 2]
```diff
@@ -18,140 +18,32 @@ import pick from 'lodash/pick';
...
-  }
-`;
+import { AccountType, Interval } from '@vegaprotocol/types';
+import { MARKET_INFO_QUERY } from './info-market-query';
 
 export interface InfoProps {
   market: MarketInfoQuery_market;
 }
 
+export const calcCandleVolume = (m: any): string | undefined => {
```
Comment (notbot00) [inline libs/deal-ticket/src/components/info-market.tsx:30]: chore (blocking): i think the market type should be generated, you don't have to use any for it

Comment (notbot00) [reply libs/deal-ticket/src/components/info-market.tsx:30]: found it:

```suggestion
export const calcCandleVolume = (m: MarketInfoQuery_market): string | undefined => {
```

**Review by notbot00 — APPROVED** [review]

File: libs/deal-ticket/src/components/info-market-query.ts [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      }
+      priceMonitoringSettings {
+        parameters {
+          triggers {
+            horizonSecs
+            probability
+            auctionExtensionSecs
+          }
+        }
+        updateFrequencySecs
```
Comment (dexturr) [inline libs/deal-ticket/src/components/info-market-query.ts:41]: chore (blocking): this field no longer exists on testnet, remove please

**Review by dexturr — CHANGES_REQUESTED** [review]
Minor ting

**Review by dexturr — APPROVED** [review]

File: libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts [hunk 4]
```diff
@@ -69,7 +69,7 @@ export interface MarketInfoQuery_market_priceMonitoringSettings_parameters_trigg
   probability: number;
   /**
    * Price monitoring auction extension duration in seconds should the price
-   * breach its theoretical level over the specified horizon at the specified
+   * breach it's theoretical level over the specified horizon at the specified
```
Comment (candida-d) [inline libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts:72]: this should be 'its', not 'it is'

File: libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts [hunk 5]
```diff
@@ -115,14 +115,52 @@ export interface MarketInfoQuery_market_data_market {
...
+   * Price monitoring projection horizon τ in seconds (> 0).
+   */
+  horizonSecs: number;
+  /**
+   * Price monitoring probability level p. (>0 and < 1)
+   */
+  probability: number;
+  /**
+   * Price monitoring auction extension duration in seconds should the price
+   * breach it's theoretical level over the specified horizon at the specified
```
Comment (candida-d) [inline libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts:130]:
```suggestion
   * breach its theoretical level over the specified horizon at the specified
```

File: libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts (hunk 4 above)
Comment (candida-d) [reply libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts:72]: Is this coming from the API?

Comment (MadalinaRaicu) [reply libs/deal-ticket/src/components/__generated__/MarketInfoQuery.ts:72]: it's coming from the API

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**